src/main/java/com/bereda/cron/ExternalExchangeRateApiScheduledTask.java

```diff
@@ -11,21 +12,15 @@
 import java.util.Map;
 
 @Component
+@AllArgsConstructor
 public class ExternalExchangeRateApiScheduledTask {
 
     private final ExternalExchangeRateApiService externalExchangeRateApiService;
     private final CurrencyService currencyService;
+    @Value("${currencies}")
     private final List<String> currenciesToImport;
 
-
-    public ExternalExchangeRateApiScheduledTask(final ExternalExchangeRateApiService externalExchangeRateApiService,
```

Comment:
if you have custom constructor - with @Value then you should create your own cosntructor. You don't have to always use lombok annotations